features/articles/Article.js

```diff
@@ -64,7 +64,10 @@ const Article = ({
   },
 }) => {
   const { brands, authors, ...tagsObject } = tagsByTopic;
-  const renderTagImage = getTagImageRenderer({ className: b('tag-image') });
+  const renderTagImage = getTagImageRenderer({
```

Comment:
Is it for those small images of authors and brands?

Reply:
yep

Reply:
Samples of both text and non-text articles having tags in viewport:

<img width="1762" alt="Screen Shot 2020-12-29 at 21 51 05" src="https://user-images.githubusercontent.com/7117196/103307081-41e70480-4a20-11eb-81cf-08a021aafd66.png">
<img width="1768" alt="Screen Shot 2020-12-29 at 21 52 41" src="https://user-images.githubusercontent.com/7117196/103307085-457a8b80-4a20-11eb-8184-9bca6d23a008.png">

So I'd make `inViewport` non-conditional.